### PR TITLE
Canairio solar version

### PR DESCRIPTION
**Pull request by roberbike**
Updated v938.
Fix GPIO35 wakeup error from Oled.
Add low CPU hz in order to save more battery
Fix GUIutils.hpp fot Tdisplay

Tested with TTGO T7 and Tdisplay.

setPowerSave for gui oled boards changued

### CODE REVIEW

File: src/power.cpp [hunk 1]
```diff
@@ -1,10 +1,14 @@
...
     digitalWrite(ADC_EN, LOW);
     delay(10);
+    rtc_gpio_init(GPIO_NUM_14);
+    rtc_gpio_set_direction(GPIO_NUM_14, RTC_GPIO_MODE_OUTPUT_ONLY);
+	rtc_gpio_set_level(GPIO_NUM_14, 1);
+    delay(500); 
+ 
 #else
    gui.PowerSave(); 
 #endif
```
Comment (hpsaturn) [inline src/power.cpp:14]: Maybe is missing the commit of this method?

**Review by hpsaturn — CHANGES_REQUESTED** [review]

File: src/power.cpp (hunk 1 above)
Comment (roberbike) [reply src/power.cpp:14]: You mean these lines of Guiutils.cpp?
72     void GUIUtils::PowerSave() {
73           u8g2.setPowerSave(1);
74     }

There is all defined, it is a function of the U8g2 library.

Comment (hpsaturn) [reply src/power.cpp:14]: ok, I didn't see that. It's ok. This class the naming of methods are with non capitalized, change PowerSave to setPowerSave like u8g2 library please

**Comment by roberbike** [comment]
Done

**Pull request merged**